### ISSUE TITLE
Bootstrap

### ISSUE THREAD

**Issue by jakee246** [jakee246/tc359finalproject#1]

**Comment by jakee246** [comment]
@chrisvfritz is this correct?

**Comment by chrisvfritz** [comment]
Not quite. You should use the latest version of `bootstrap-sass` in your Gemfile, then look at [their README](https://github.com/twbs/bootstrap-sass#a-ruby-on-rails) for instructions on integrating it into Rails.

**Comment by jakee246** [comment]
@chrisvfritz It should be correct now.

**Comment by chrisvfritz** [comment]
Better, but Bootstrap's JavaScript still isn't being included in the app. The last part of the Ruby on Rails section in those docs has instructions for how to include it.

**Comment by jakee246** [comment]
Now we should be good to go.

**Comment by chrisvfritz** [comment]
Unfortunately, that isn't the right way to include Bootstrap's JS via  the `bootstrap-sass` gem. With your way, updating the gem will still leave you with an old version of Bootstrap's JS.

The third-to-last paragraph in [these docs](https://github.com/twbs/bootstrap-sass#a-ruby-on-rails) describes how to properly include it.

**Comment by jakee246** [comment]
@chrisvfritz never received feedback on this. Thank you.

**Comment by chrisvfritz** [comment]
Ah, I only know to take another look when you've posted a comment mentioning me. You still need to delete [your manually added version of Bootstrap's JS here](https://github.com/jakee246/tc359finalproject/blob/master/app/assets/javascripts/bootstrap.js). Then in your application.js file, [this line](https://github.com/jakee246/tc359finalproject/blob/master/app/assets/javascripts/application.js#L17) should be above `//= require_tree .`. For future reference, `require_tree` adds all the JavaScript in your javascripts folder, so 3rd-party assets should _always_ appear above the `require_tree`, so that you can access those libraries from your own code.

**Comment by jakee246** [comment]
Fixed.

**Comment by jakee246** [comment]
@chrisvfritz fixed

**Comment by chrisvfritz** [comment]
This looks good now, but it was posted past the deadline on Friday, so I'm not able to give you credit for it unfortunately. Sorry. :disappointed:

**Comment by jakee246** [comment]
We went through these together in case on that Monday and I had thought
gotten the green light from you before I left. My confusion is holding me
from obtaining a 3.5 now correct?

**Comment by chrisvfritz** [comment]
Ah, I didn't remember that we went through them on Monday. I just looked at the commit time. I'm happy to award the points in that case. :+1:

**Comment by jakee246** [comment]
Thank you very much!!
